eZ/Publish/API/Repository/URLService.php

```diff
@@ -20,17 +24,16 @@ interface URLService
      *
      * @return \eZ\Publish\API\Repository\Values\URL\URLUpdateStruct
      */
-    public function createUpdateStruct();
+    public function createUpdateStruct(): URLUpdateStruct;
 
     /**
      * Find URLs.
      *
-     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
```

Comment:
Should we delete this?

Reply:
Good catch @mikadamczyk. Restored in https://github.com/ezsystems/ezpublish-kernel/pull/2854/commits/3ec650a651309cd5e0500b11bb3fa3e760902084